### ISSUE TITLE
Simplify set workspace command

### ISSUE THREAD

**Issue by tedmiston** [astronomerio/astro-cli#89]
With the CLI it would be cool from a UX perspective if I could do:

    $ astro workspace set <workspace-name>

Instead of:

    $ astro config set project.workspace <workspace-id>

As a user, it feels like choosing my active workspace naturally falls under the workspace command group.

This was first pain point I hit having multiple workspaces and doing a deploy.

Note: This would require making workspace names unique as they currently are not.

Also, it would be nice to have the CLI still default to the default workspace even when multiple have been created (vs defaulting to none which I think it is doing currently).

What do others think (especially if you've already tried multiple workspaces today)? @andscoop @schnie @ryw @cwurtz

**Comment by andscoop** [comment]
@tedmiston as you suggested we would need to make workspace names unique. There also is currently no meta data around which workspace is default. We used to name it "Personal" so we could fake it. 

Other than those two changes that need to be made in houston and the data model - I'm all for it.

**Comment by tedmiston** [comment]
Good point maybe we could do something like how `cd` with no path goes to your home directory, like `astro workspace set` would change back to default.

**Comment by andscoop** [comment]
This will become much simpler once #96 is in place

**Comment by andscoop** [comment]
- `astro workspace list`  to get workspaces
- `astro workspace switch [WORKSPACEUUID]` to switch
